src/Logging.AzureAppServices/src/BlobLoggerProvider.cs

```diff
@@ -77,7 +77,7 @@ internal override async Task WriteMessagesAsync(IEnumerable<LogMessage> messages
 
                     await writer.FlushAsync();
                     var tryGetBuffer = stream.TryGetBuffer(out var buffer);
-                    Debug.Assert(tryGetBuffer);
+                    System.Diagnostics.Debug.Assert(tryGetBuffer);
```

Comment:
There's a using for `System.Diagnostics`. Is `Debug` now ambiguous when building for `net6.0`❔ Don't do this if not.

Reply:
Without this we get:

> error CS0234: The type or namespace name 'Assert' does not exist in the namespace 'Microsoft.Extensions.Logging.Debug'

There are a couple other places in the repo where we do this, e.g. https://github.com/dotnet/aspnetcore/blob/7e145b1132ebe0bf85765f5ef96d337ff2d790e2/src/Http/Http/src/Features/FormFeature.cs#L262